lib/ansible/modules/cloud/amazon/elb_application_lb.py

```diff
@@ -538,9 +538,13 @@ def compare_listener(current_listener, new_listener):
         if current_listener['SslPolicy'] != new_listener['SslPolicy']:
             modified_listener['SslPolicy'] = new_listener['SslPolicy']
         if current_listener['Certificates'][0]['CertificateArn'] != new_listener['Certificates'][0]['CertificateArn']:
+            modified_listener['Certificates'] = []
+            modified_listener['Certificates'].append({})
```

Comment:
Can you just do something like `modified_listener['Certificates'] = [{'CertificateArn': new_listener['Certificates'][0]['CertificateArn']}]` rather than making it an empty list and appending an empty dict?

Reply:
I see that it's the same as done elsewhere in the module though. Fine as-is.